docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer talk to the Viam cloud through [the Viam app](https://app.viam.com).
+
+### How does `viam-server` work if your robot loses connectivity during a session?
+
+If the robot loses connectivity, `viam-server` will timeout and end any client [*sessions*](/program/apis/sessions/) on this robot.
+When a session times out, any active commands will be cancelled, stopping any moving parts, and no new commands will be able to reach the robot until the connection is restored.
+
+To disable the default behavior here and manage resource timeout and reconfiguration over a networking session yourself, follow [these instructions](/program/apis/sessions/).
+[Disable the default behavior](/program/apis/sessions/#disable-default-session-management) of session management, then utilize [Viam's SDKs](/program/) in your code to make calls to [the session management API](https://pkg.go.dev/go.viam.com/rdk/session#hdr-API).
+
+If you do this, you can manage the network operations of your robot manually, and operate your robot without the need for a consistent network connection.
+However, the low-network approach is not recommended if you want to utilize any of Viam's [services](/services/), like the vision service, motion planning, or the frame system.
```

Comment:
This line probably needs to be removed if the above line is as well. Unless you want to completely rethink this and explain how to configure non-cloud connected robots (but I think it was decided that was out of scope for this page.)